keyboards/edi/hardlight/Mk1/hardlight.c

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
These two files need to be named `mk1.[ch]`, and the folder name needs to be lowercase.

Reply:
Right, right. Done!